### PR TITLE
CIFUZZ-528: add action to list findings as github checks

### PR DESCRIPTION
**Pull request by m-pongratz**

### CODE REVIEW

**Review by adombeck — COMMENTED** [review]
Code LGTM. I wasn't able to test the action yet. I tried [here](https://github.com/ci-fuzz/zint/pull/49) but the fuzzing run didn't find anything. Will look into again tomorrow.

File: list-findings/action.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: 'List findings'
+description: 'List the found findings of a fuzzing run'
+inputs:
+  ci-fuzz-api-token:
+    description: 'CI Fuzz Access Token'
+    required: true
+  github-token:
+    description: 'Github Access Token'
```
[the remark this answers is not in the document]
Comment (m-pongratz) [reply list-findings/action.yml]: fixed

File: list-findings/action.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    required: true
+  serverUrl:
+    description: 'Server Url for fuzz testing'
+    required: true
+  projectId:
+    description: 'Id of the project'
```
[the remark this answers is not in the document]
Comment (m-pongratz) [reply list-findings/action.yml]: fixed

File: list-findings/action.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+description: 'List the found findings of a fuzzing run'
+inputs:
+  ci-fuzz-api-token:
+    description: 'CI Fuzz Access Token'
+    required: true
+  github-token:
+    description: 'Github Access Token'
+    required: true
+  serverUrl:
+    description: 'Server Url for fuzz testing'
```
[the remark this answers is not in the document]
Comment (m-pongratz) [reply list-findings/action.yml]: fixed

File: list-findings/action.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    required: true
+  projectId:
```
[the remark this answers is not in the document]
Comment (m-pongratz) [reply list-findings/action.yml]: We changed the variablenames to fit the names of the previous actions.

**Comment by adombeck** [comment]
Did you configure any project to use this GitHub Action, so that I can look at what the results look like?

**Comment by m-pongratz** [comment]
Hi Adrian, here you can see the triggered action:
https://github.com/CC-CIFUZZ-TEST/lighttpd/pull/15

**Pull request merged**